components/sync15/src/request.rs

```diff
@@ -921,6 +928,7 @@ mod test {
         );
     }
 
+    // Testing if a single batch works
```

Comment:
ditto, here and below.

In general, the comment has to tell me more than this to be useful. Typically, for unit tests, the test name should be descriptive enough in order to convey what the test does, and therefore there is usually less comments on tests.

Compare "Testing if a single batch works" with "Validate that all queued records are posted when flushing succeeds."